### PR TITLE
added scraper for Offenbach and Kreis Offenbach

### PR DESCRIPTION
**Pull request by fbartnitzek**
added scraper for Offenbach and Kreis Offenbach

### CODE REVIEW

File: landkreise/get-offenbach-kreis.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# print("Todesfall: ", m.group(3))
+# print("Stand: ", m.group(4))
+
+cases = int(m.group(1))
+status = get_status(m.group(4))
+
+# print("cases: ", cases)
+# print("status: ", status)
+
+add_to_database("06438000", status, cases, "Kreis Offenbach")
```
Comment (dadosch) [inline landkreise/get-offenbach-kreis.py:50]: Is there a reason for the trailing zeros?

Wikipedia shows 06438  as the AGS; our csv at https://raw.githubusercontent.com/corona-zahlen-landkreis/corona_landkreis_fallzahlen_scraping/895afda1da29f1f00c2845617effdcd0011a469a/sources.csv also shows 06438

**Comment by dadosch** [comment]
You can also raise the counter in the readme by 2, if you mind.

File: landkreise/get-offenbach-kreis.py (hunk 1 above)
Comment (fbartnitzek) [reply landkreise/get-offenbach-kreis.py:50]: no, but some other scraper included them...

Comment (dadosch) [reply landkreise/get-offenbach-kreis.py:50]: Is the AGS wrong for any of them? I just checked a few (dresden, hamburg, Kassel) and it all matched with the wikipedia entry :smiley:

**Comment by dadosch** [comment]
Thank you :)

**Pull request merged**